### PR TITLE
Adds preference for "Tagger" paint color.

### PR DESCRIPTION
**Pull request by lizardqueenlexi**
## About The Pull Request

Per the title, this PR allows you to pick your starting paint color from the "Tagger" quirk on the character preferences menu.

image

This replaces the starting color being random; it does not prevent you from changing the color later as normal.
## Why It's Good For The Game

It's a minor quality of life change. This will mostly be helpful to players who have some "signature" color they like to use, to prevent having to manually select it (and possibly input a color code) every round. It will be of less relevance to those who tend to select new colors every round anyway.

Possible downsides are mainly adding another pref to the menu, although this shouldn't be too much of an annoyance since it only appears if you already have the relevant quirk. It does also remove the _ability_ to have a randomly-chosen paint color, though I'm not sure if that matters.
## Changelog
:cl:
qol: you can choose your default paint color for the "Tagger" quirk from prefs.
/:cl:

### CODE REVIEW

File: code/modules/client/preferences/paint_color.dm [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/datum/preference/color/paint_color
+	savefile_key = "paint_color"
+	savefile_identifier = PREFERENCE_CHARACTER
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+
+/datum/preference/color/paint_color/is_accessible(datum/preferences/preferences)
+	if (!..(preferences))
+		return FALSE
```
Comment (san7890) [inline code/modules/client/preferences/paint_color.dm:8]: We don't mutate `preferences` at all, so this doesn't seem necessary... Shouldn't the following work?

```suggestion
/datum/preference/color/paint_color/is_accessible(datum/preferences/preferences)
	. = ..()
	if (!.)
		return FALSE
```

Calls to parent automatically feed in the arglist iirc.

Comment (lizardqueenlexi) [reply code/modules/client/preferences/paint_color.dm]: That does seem to work, thank you.

Comment (Mothblocks) [reply code/modules/client/preferences/paint_color.dm]: They all do it for explicitness and because we never benefit from `.` here, plz dont do this change

File: code/modules/client/preferences/paint_color.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/datum/preference/color/paint_color
+	savefile_key = "paint_color"
+	savefile_identifier = PREFERENCE_CHARACTER
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+
+/datum/preference/color/paint_color/is_accessible(datum/preferences/preferences)
+	. = ..()
+	if (!.)
+		return FALSE
```
Comment (Mothblocks) [inline code/modules/client/preferences/paint_color.dm:9]: This was fine as is and I think you should revert the change

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/client/preferences/paint_color.dm [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/datum/preference/color/paint_color
```
Comment (Mothblocks) [inline code/modules/client/preferences/paint_color.dm:1]: Auto-doc this

File: code/modules/client/preferences/paint_color.dm (hunk 1 above)
Comment (san7890) [reply code/modules/client/preferences/paint_color.dm]: I think it was done as its own commit so shouldn't be hard to revert
